libs/ui/src/consts.ts

```diff
@@ -67,3 +68,9 @@ export const Color = {
   neutral10: '#23191A',
   neutral0: '#000000',
 }
+
+document.fonts.ready.then(() => {
+  if (document.fonts.check('1em studiofeixenmono')) {
+    document.documentElement.classList.add('fonts-loaded')
+  }
+})
```

Comment:
This seems odd.
Every time this file is imported it'll attach that event handler, no?

Reply:
Oops! That was to debug why fonts were not loading, removed/cleaned that up now.